include/problem_operators/problem_operator_interface.h

```diff
@@ -22,6 +22,11 @@ class ProblemOperatorInterface
   // Reference to the current problem.
   platypus::Problem & _problem;
 
+  // Vector of names of state gridfunctions used in formulation, ordered by appearance in block
+  // vector during solve.
+  std::vector<std::string> _test_var_names;
```

Comment:
Why were these variables places here in the interface class? Just asking because this will affect the new equation system structure, where members like these would normally be placed in the EquasionSystemData class.